idd/Energy+.idd.in

```diff
@@ -63383,12 +63383,15 @@ Boiler:Steam,
        \key OtherFuel1
        \key OtherFuel2
   N1 , \field Maximum Operating Pressure
-       \units Kpa
+       \required-field
+       \units Pa
   N2 , \field Theoretical Efficiency
```

Comment:
Simply changing the IDD units is dangerous.  Seems there should be a code change or transition rule to check the order of magnitude and warn if it looks like the value is in kPa instead of Pa.